detection_rules/__init__.py

```diff
@@ -41,3 +44,8 @@
     'schemas',
     'utils'
 )
+
+try:
+    __version__ = get_distribution(__name__).version
```

Comment:
We need to replace this with a proper version (and discuss our versioning approach).

This is the version it generates currently
`detection-rules @ file:///full/path/to/detection-rules`

Reply:
Really? When I install I get something different. 

```bash
Building wheels for collected packages: detection-rules
  Building wheel for detection-rules (pyproject.toml) ... done
  Created wheel for detection-rules: filename=detection_rules-8.3.0-py3-none-any.whl size=11466244 sha256=50af0686e530da43bf3705c25115c49c65b73e7e60fecb5308e9431f56d73b3c
  Stored in directory: /Users/stryker/Library/Caches/pip/wheels/ab/10/49/e4e6c8bea78201b2fff831580575ff466e48abed975357b01f
Successfully built detection-rules
Installing collected packages: detection-rules
Successfully installed detection-rules-8.3.0
```

It pulls from the setup.cfg which is the convention.

Reply:
try `pip freeze`

Reply:
Looking deeper we saw this to occur with a pip version >20.1.